### PR TITLE
[hma] Make build work for windows

### PR DESCRIPTION
**Pull request by Dcallies**
Summary
---------

Prettier doesn't seem to like windows, even with LF mode enabled for git checkout.

Test Plan
---------

npm run-script build

### CODE REVIEW

File: hasher-matcher-actioner/webapp/.eslintrc.json [hunk 1]
```diff
@@ -24,7 +24,8 @@
     "rules": {
         "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
         "prettier/prettier": [
-            "error"
+            "error",
+			{"endOfLine":"auto"}
```
Comment (schatten) [inline hasher-matcher-actioner/webapp/.eslintrc.json:28]: nit: tab/space thingy.

**Review by schatten — APPROVED** [review]
Approve to unblock. Small nit about tab/space. LGTM otherwise.

**Review by BarrettOlson — APPROVED** [review]

File: hasher-matcher-actioner/webapp/.eslintrc.json (hunk 1 above)
Comment (Dcallies) [reply hasher-matcher-actioner/webapp/.eslintrc.json:28]: Hmm, weird, used notepad++

Comment (Dcallies) [inline hasher-matcher-actioner/webapp/.eslintrc.json:28]:
```suggestion
            "error",
            {"endOfLine":"auto"}
```

File: hasher-matcher-actioner/webapp/.prettierrc.json [hunk 2]
```diff
@@ -4,6 +4,7 @@
     "trailingComma": "all",
     "bracketSpacing": false,
     "jsxBracketSameLine": true,
+	"endOfLine":"auto",
     "overrides": [
```
Comment (Dcallies) [inline hasher-matcher-actioner/webapp/.prettierrc.json:8]:
```suggestion
    "jsxBracketSameLine": true,
    "endOfLine":"auto",
    "overrides": [
```

**Pull request merged**